linux-bore/auto-cpu-optimization.sh

```diff
@@ -29,6 +29,10 @@ elif [[ ${MARCH} == "GOLDMONT-PLUS" ]]; then
     MARCH="GOLDMONTPLUS"
 elif [[ ${MARCH} == "SKYLAKE-AVX512" ]]; then
     MARCH="SKYLAKE2"
+elif [[ ${MARCH} == "MIVYBRIDGE" ]]; then
+    scripts/config --disable CONFIG_AGP_AMD64 
```

Comment:
As we already wrote privately we could probably add this for every INTEL based CPU. 
Or we will ad it at the configure script to MNATIVE_INTEL / MNATIVE_AMD. 

All in all it would be nice if you can test it, since so far I know you have a INTEL CPU

Reply:
Seems good